src/components/logo/Logo.stories.mdx

```diff
@@ -0,0 +1,64 @@
+import {ArgsTable, Meta, Story, Canvas} from '@storybook/addon-docs';
+import {StoryVariant} from '../../_docs/utils';
+import {Preview} from '../../_docs/blocks/';
+
+import Logo, {TYPE} from './Logo';
+import LogoA11y from './stories/Logo.a11y.mdx';
+
+<Meta title="Components/Logo" component={Logo} />
+
+# Logo
+
+Table of contents:
+
+- [Props](#props)
+- [Stories](#stories)
+- [Accesibility](#accessibility)
+
+## Props
```

Comment:
This shouldn't be a separate section. Let's put args table directly under default story